keyboards/handwired/ms_sculpt_mobile/8x18_arm/matrix.c

```diff
@@ -0,0 +1,338 @@
+ /*
```

Comment:
Is there a reason for the custom matrix here?

Reply:
Custom matrix is not required on f103, the only part that jumps out is the stuff in `matrix_init` which could be achieved with `keyboard_pre_init_kb`/ `keyboard_post_init_kb`.

Reply:
removed as suggested